src/lib/libemval.js

```diff
@@ -305,26 +271,38 @@ var LibraryEmVal = {
 
 #if !DYNAMIC_EXECUTION
     var argN = new Array(argCount);
-    var invokerFunction = (obj, func, destructorsRef, args) => {
+    var invokerFunction = (handle, methodName, destructorsRef, args) => {
```

Comment:
Would it make more sense to call the first argument `receiver`?  Or is `handle` (the type of the receiver I guess?) more descriptive here?

Reply:
`handle` as in generally `EM_VAL`. If the invoker kind is "method" then it's a handle of the receiver, if it's a "function" or a "constructor" then it's a handle of the function itself, and if it's a cast, then it's just null... A handle seemed like the most generic name.